### PR TITLE
use SleepFor() instead of usleep()

### PR DESCRIPTION
**Pull request by vagetablechicken**
https://github.com/apache/incubator-doris/issues/3204#issuecomment-604334778

contains so many reformat diffs, please use commit filter, or I can revert the format diffs.

### CODE REVIEW

File: be/src/exprs/utility_functions.cpp [hunk 1]
```diff
@@ -17,27 +17,25 @@
...
 }
 
-BooleanVal UtilityFunctions::sleep(
-        FunctionContext* ctx, const IntVal& milliseconds) {
+BooleanVal UtilityFunctions::sleep(FunctionContext* ctx, const IntVal& milliseconds) {
     if (milliseconds.is_null) {
         return BooleanVal::null();
     }
-    usleep(milliseconds.val * 1000 * 1000);
+    usleep(milliseconds.val * 1000 * 1000); // TODO(hw): ms * 1000 * 1000 = ns
```
Comment (imay) [inline be/src/exprs/utility_functions.cpp]: why not change this?

Comment (vagetablechicken) [reply be/src/exprs/utility_functions.cpp]: I'm confused about this. This usleep() use ns as its argument. Is it a mistake?
I should change it to
`SleepFor(MonoDelta::FromMilliseconds(milliseconds.val)` ?

Comment (imay) [reply be/src/exprs/utility_functions.cpp]: this function is used to support SQL `select sleep(N)`; N is for N second. so you just change it to `sleepFor(Second)`

File: be/src/agent/task_worker_pool.cpp [hunk 2]
```diff
@@ -668,8 +658,8 @@ void* TaskWorkerPool::_push_worker_thread_callback(void* arg_this) {
             error_msgs.push_back("push request push_type invalid.");
             task_status.__set_status_code(TStatusCode::ANALYSIS_ERROR);
         } else {
-            OLAP_LOG_WARNING("push failed, error_code: %d, signature: %ld",
-                             status, agent_task_req.signature);
+            OLAP_LOG_WARNING("push failed, error_code: %d, signature: %ld", status,
```
Comment (chaoyli) [inline be/src/agent/task_worker_pool.cpp]: use LOG(WARNING) instead

File: be/src/agent/task_worker_pool.cpp [hunk 3]
```diff
@@ -718,11 +708,12 @@ void* TaskWorkerPool::_publish_version_worker_thread_callback(void* arg_this) {
             if (res == OLAP_SUCCESS) {
                 break;
             } else {
-                OLAP_LOG_WARNING("publish version error, retry. "
-                                 "[transaction_id=%ld, error_tablets_size=%d]",
-                                 publish_version_req.transaction_id, error_tablet_ids.size());
+                OLAP_LOG_WARNING(
```
Comment (chaoyli) [inline be/src/agent/task_worker_pool.cpp]: use LOG(WARNING) instead

File: be/src/agent/task_worker_pool.cpp [hunk 4]
```diff
@@ -967,8 +956,8 @@ void* TaskWorkerPool::_storage_medium_migrate_worker_thread_callback(void* arg_t
         EngineStorageMigrationTask engine_task(storage_medium_migrate_req);
         OLAPStatus res = worker_pool_this->_env->storage_engine()->execute_task(&engine_task);
         if (res != OLAP_SUCCESS) {
-            OLAP_LOG_WARNING("storage media migrate failed. status: %d, signature: %ld",
-                             res, agent_task_req.signature);
+            OLAP_LOG_WARNING("storage media migrate failed. status: %d, signature: %ld", res,
```
Comment (chaoyli) [inline be/src/agent/task_worker_pool.cpp]: use LOG(WARNING) instead

File: be/src/agent/task_worker_pool.cpp [hunk 5]
```diff
@@ -1016,20 +1005,17 @@ void* TaskWorkerPool::_check_consistency_worker_thread_callback(void* arg_this)
...
-                check_consistency_req.version_hash,
-                &checksum);
+        EngineChecksumTask engine_task(
+                check_consistency_req.tablet_id, check_consistency_req.schema_hash,
+                check_consistency_req.version, check_consistency_req.version_hash, &checksum);
         OLAPStatus res = worker_pool_this->_env->storage_engine()->execute_task(&engine_task);
         if (res != OLAP_SUCCESS) {
-            OLAP_LOG_WARNING("check consistency failed. status: %d, signature: %ld",
-                             res, agent_task_req.signature);
+            OLAP_LOG_WARNING("check consistency failed. status: %d, signature: %ld", res,
```
Comment (chaoyli) [inline be/src/agent/task_worker_pool.cpp]: use LOG(WARNING) instead

File: be/src/agent/task_worker_pool.cpp [hunk 6]
```diff
@@ -1169,7 +1160,8 @@ void* TaskWorkerPool::_report_tablet_worker_thread_callback(void* arg_this) {
 
         request.__set_report_version(_s_report_version);
         OLAPStatus report_all_tablets_info_status =
-                StorageEngine::instance()->tablet_manager()->report_all_tablets_info(&request.tablets);
+                StorageEngine::instance()->tablet_manager()->report_all_tablets_info(
+                        &request.tablets);
         if (report_all_tablets_info_status != OLAP_SUCCESS) {
             OLAP_LOG_WARNING("report get all tablets info failed. status: %d",
```
Comment (chaoyli) [inline be/src/agent/task_worker_pool.cpp]: use LOG(WARNING) instead

**Review by chaoyli — CHANGES_REQUESTED** [review]

**Review by imay — APPROVED** [review]
LGTM

**Review by chaoyli — APPROVED** [review]

**Pull request merged**